### ISSUE TITLE
AnimateHeightProps no longer exported in 3.0 (typescript)

### ISSUE THREAD

**Issue by xec** [Stanko/react-animate-height#133]
**Code example**
```ts
import AnimateHeight, { AnimateHeightProps } from 'react-animate-height'
```

Works fine in 2.* but in 3.0 results in error message:

```
 error TS2614: Module '"react-animate-height"' has no exported member 'AnimateHeightProps'. Did you mean to use 'import AnimateHeightProps from "react-animate-height"' instead?
 ```

**Expected behavior**
Can import the props definition as in 2.*

**Possible Solution**
The AnimateHeightProps is defined in the published typescript definition file, but is missing the `export` keyword
Please export the `AnimateHeightProps` typescript interface :)

**Your Environment**
 - Version used: 3.0.4
 - OS: windows
 - Browser n/a (project doesn't compile)
 - Version n/a

**Additional context**
I'm writing react components for a design system, and since animating height in css is complicated, we've created a SlideDown component that is basically just a wrapper for AnimateHeight with some styling applied, with a few props added on.

In order to support (and forward) the same props that AnimateHeight does, it is very useful for us to be able to extend the props type.

**Comment by Stanko** [comment]
Thanks, fixed in v3.0.5

**Comment by xec** [comment]
Wow, that was quick! Thank you! :)
